exsrv.cmd

```diff
@@ -0,0 +1,71 @@
+:: Utility Batch File To Start/Stop Elixir App As Windows Service
+:: Onorio Catenacci
+:: 19 November 2015
+
+@echo off
+SET THISFILENAME=%~n0
+SET COMMAND=%2
+SET ERTS_BIN_PATH=C:/progra~1/erl7.1/erts-7.1/bin
```

Comment:
Isn't the purpose of using exrm to not have to install erlang and all when deploying? Wouldn't this need to be relative to the release, and specific to the version listed in the data file?